src/main/java/net/dv8tion/jda/core/EmbedBuilder.java

```diff
@@ -460,6 +498,17 @@ public EmbedBuilder addBlankField(boolean inline)
         return this;
     }
     
+    /**
+     * Clears all fields from the embed.
+     *
+     * @return the builder after the field has been added
+     */
```

Comment:
This is a bit ambiguous. Please somehow link this to __#addField__ and explain what you exactly mean with "fields"